salt/ufw/etc_ufw/sysctl.conf

```diff
@@ -32,10 +56,42 @@ net/ipv4/conf/default/log_martians=0
 #net/ipv4/tcp_fin_timeout=30
 #net/ipv4/tcp_keepalive_intvl=1800
 
+# Ignore Directed pings
+#net/ipv4/icmp_echo_ignore_all = 1
```

Comment:
Yes, at line 49 we are allowing ICMP echo. However, if someone wants to blocka all echo should uncomment this line.